pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -2176,7 +2176,7 @@ func TestPreempt(t *testing.T) {
 				schedulerapi.DefaultPercentageOfNodesToScore,
 				true)
 			state := framework.NewCycleState()
-			scheduler.(*genericScheduler).snapshot()
+			scheduler.(*genericScheduler).Snapshot()
```

Comment:
No need to cast if it's part of the interface.

Reply:
done.